tests/whitebox/integration/logical/test_rename.py

```diff
@@ -19,11 +19,13 @@
 from dbus_client_gen import DbusClientUniqueResultError
 
 # isort: LOCAL
+from stratis_cli import StratisCliErrorCodes
 from stratis_cli._errors import StratisCliActionError, StratisCliNoChangeError
 
-from .._misc import RUNNER, SimTestCase, device_name_list
+from .._misc import RUNNER, SimTestCase, check_error, device_name_list
 
 _DEVICE_STRATEGY = device_name_list(1)
+ERROR = StratisCliErrorCodes.ERROR
```

Comment:
Please prepend underscores for your ```ERROR``` constants in every file, like ```_ERROR```. This indicates that they are intended to be constants that are used only locally, in the file in which they are defned.